templates/component.html

```diff
@@ -2,34 +2,39 @@
 layout: base
 ---
 <div class="content-banner">
-  <div class="hxRow">
-    <div class="hxCol-8">
-      <div class="hxRow">
-        <div class="hxCol">
-          <div class="component-title">{{page.title}}</div>
+    <div class="content-wrapper">
```

Comment:
Did you also make these changes to the component-code template, or are we skipping that page for now? I don't see it in this PR, but maybe it happened in a separate one?

Reply:
I didn't.  We need to figure out the relationship between the Helix website and the helix-ui site.  So, for right now, I'm holding off on the component code pages.